README.md

```diff
@@ -47,4 +47,23 @@ When adding new test workflows, please follow these guidelines:
 - Document runtime requirements clearly
 - Isolate it to its own directory
 
+## Cromwell tests
 
+A suite of tests for running WDLs against Cromwell are located in `tests/cromwell-api`. Tests are run using Python. To run tests:
+
+- clone this repository locally and cd into it
+- if you don't have `uv` installed, [install it][uvinstall]
+- create a virtual environment: `uv venv`. This should pick up the Python version specified in `.python-version`. If you do not have the version given in `.python-version` it will be installed by `uv`. 
+- activate the virtual environment: `source .venv/bin/activate`
+- check that you have the Python version in `.python-version`: `python --version`
+- set a PROOF token for the test user used for these tests as `PROOF_API_TOKEN_DEV`. the value is in 1password - talk to Sean or Scott if you don't have access
```

Comment:
What's the actual best practice for doing this? Is there any secret sauce to tying these docs in with python or uv, or is that not necessary? https://developer.1password.com/docs/cli/secrets-environment-variables/

Reply:
I considered suggesting the 1password cli, but figured that was more complexity, but could suggest a few options there